components/docs-chef-io/content/automate/ha_chef_backend_to_automate_ha.md

```diff
@@ -17,276 +15,368 @@ gh_repo = "automate"
 {{< /warning >}}
 
 {{< warning >}}
+
 - Customers using only **Standalone Chef Infra Server** or **Chef Backend** are advised to follow this migration guidance. Customers using **Chef Manage** or **Private Chef Supermarket** with Chef Backend should not migrate with this.
-- Also, for the customers using a standalone Chef Infra Server, cookbooks should be in the database but not in either the file system or S3.
-- Automate HA do not support the super market authentication with chef-server users credentials. 
-- Post Migration Customer can not login with chef-server users to Supermarket. 
+- Also, for the customers using a standalone Chef Infra Server, cookbooks should be in the database or S3 but not in the file system.
+- Automate HA does not support supermarket authentication with chef-server user credentials.
+- Post Migration Customer can not log in with chef-server users to Supermarket.
+
 {{< /warning >}}
 
 This page explains the procedure to migrate the existing Standalone Chef Infra Server or Chef Backend data to the newly deployed Chef Automate HA. This migration involves two steps:
 
--   Back up the data from an existing Chef Infra Server or Chef Backend via `knife-ec-backup`.
--   Restore the backed-up data to the newly deployed Chef Automate HA environment via `knife-ec-restore`.
+- Back up the data from an existing Chef Infra Server or Chef Backend via `knife-ec-backup`.
+- Restore the backed-up data to the newly deployed Chef Automate HA environment via `knife-ec-restore`.
 
-Take backup using the `knife-ec-backup` utility and move the backup folder to the newly deployed Chef Server. Later, restore using the same utility. The backup migrates all the cookbooks, users, data-bags, policies, and organizations.
-
-`knife-ec-backup` utility backups and restores the data in an Enterprise Chef Server installation, preserving the data in an intermediate, editable text format. It is similar to the knife download and knife upload commands and uses the same underlying libraries. It includes workarounds for unsupported objects by the tools and various Server API deficiencies. The goal is to improve knife download, knife upload, and the Chef Infra Server API to deprecate the tool.
+Take a backup using the `knife-ec-backup` utility and move the backup folder to the newly deployed Chef Server. Later, restore using the same utility. The backup migrates all the cookbooks, users, data bags, policies, and organizations. `knife-ec-backup` utility backups and restores the data in an Enterprise Chef Server installation, preserving the data in an intermediate, editable text format. It is similar to the knife download, uploads commands, and uses the same underlying libraries. It includes workarounds for unsupported objects by the tools and various Server API deficiencies. The goal is to improve knife download, knife upload, and the Chef Infra Server API to deprecate the tool.
```

Comment:
Why is this needed ?
`It is similar to the knife download, uploads commands, and uses the same underlying libraries. It includes workarounds for unsupported objects by the tools and various Server API deficiencies. The goal is to improve knife download, knife upload, and the Chef Infra Server API to deprecate the tool.`
I feel we can remove this.
```suggestion
Take a backup using the `knife-ec-backup` utility and move the backup folder to the newly deployed Chef Server. Later, restore using the same utility. The backup migrates all the cookbooks, users, data bags, policies, and organizations. `knife-ec-backup` utility backups and restores the data in an Enterprise Chef Server installation, preserving the data in an intermediate, in editable text format.
```